readme.txt

```diff
@@ -194,6 +194,78 @@ Yes, through the addition of one or more of the add-on payment gateways, you can
 
 == Changelog ==
 
+= 2.7, Date To Be Determined =
+* New: Introduced the EDD_Cart class, for managing the shopping cart contents.
+* New: Modifying price, quantity, and tax of individual items when viewing a payment is now possible.
```

Comment:
Let's expand this to include a mention of the redesign of the entire section, not just the new features.